comm.py

```diff
@@ -89,12 +89,32 @@ def send_to_simulator(msg):
     except Exception as e:
         print('Unexpected error occured:', e)
 
+def convert_command_syntax(cmd: str):
+    """
+    Converts website command syntax to ground station software's syntax.
+
+    Currently, the website's command syntax is:
+        `command.name arg1 arg2 ...`
+    but ground station software's command syntax is:
+        `command.name(arg1 arg2 ...)`
 
-def send_to_satellite(sock, csp, msg):
+    TODO: Eventually, change the "Live Commands" syntax on the website
+          to match ground station software's for consistency.
```

Comment:
```suggestion
```

- [ ] Lets create an issue out of this.
- [ ] Add a brief description of what the function does.